react-16.8-test-site/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
why do we need this? this would suppress any errors with the project dependency tree right?

Reply:
Yeah, it's not happy about the newer version of babel higher up in the tree. I didn't really see a good way around this, we technically don't need the test-site I just thought it might be easier for other people

Reply:
oh hmm, I wonder if doing `overrides` for babel version would help in test site. If not, this sounds fine!